cmd/clusterctl/client/config/reader_viper_test.go

```diff
@@ -156,6 +158,22 @@ func Test_viperReader_Get(t *testing.T) {
 	}
 }
 
+func Test_viperReader_GetWithoutDefaultConfig(t *testing.T) {
+	g := NewWithT(t)
+	dir, err := ioutil.TempDir("", "clusterctl")
+	g.Expect(err).NotTo(HaveOccurred())
+	defer os.RemoveAll(dir)
```

Comment:
This directory is not needed here since the `viperReader` will have no `configPaths` for this test case.

Reply:
thanks for noticing this. now I'm using it so the test is isolated from the user config